### PR TITLE
docs: Adjust docs for the wizard

### PR DESCRIPTION
**Pull request by jeromelaban**

### CODE REVIEW

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.1

File: doc/articles/create-an-app-rider.md [hunk 1]
```diff
@@ -12,17 +12,32 @@ uid: Uno.GettingStarted.CreateAnApp.Rider
...
+1. Configure your new project by providing a project name, then click **Start**
+
+    ![A screen showing how to choose a solution name](Assets/quick-start/live-wizard-01-choose-name.png)
+
+1. Choose a template to build your application
+
+    ![A screen showing how to choose template for the new app](Assets/quick-start/live-wizard-02-select-preset.png)
+
+    > [!TIP]
+    > For a detailed overview of the Uno Platform project template wizard and all its options, see [this guide](xref:Uno.GettingStarted.UsingWizard).
```
Comment (Youssef1313) [inline doc/articles/create-an-app-rider.md:27]: https://github.com/unoplatform/uno/issues/15048

```suggestion
    > For a detailed overview of the Uno Platform project template wizard and all its options, see the [Template Wizard guide](xref:Uno.GettingStarted.UsingWizard).
```

File: doc/articles/create-an-app-vscode.md [hunk 2]
```diff
@@ -8,19 +8,29 @@ uid: Uno.GettingStarted.CreateAnApp.VSCode
...
-> [!TIP]
-> `MyApp` is the name you want to give to your project.
+    ![A screen showing how to choose a solution name](Assets/quick-start/live-wizard-01-choose-name.png)
+
+1. Choose a template to build your application
+
+    ![A screen showing how to choose template for the new app](Assets/quick-start/live-wizard-02-select-preset.png)
+
+    > [!TIP]
+    > For a detailed overview of the Uno Platform project template wizard and all its options, see [this guide](xref:Uno.GettingStarted.UsingWizard).
```
Comment (Youssef1313) [inline doc/articles/create-an-app-vscode.md:23]:
```suggestion
    > For a detailed overview of the Uno Platform project template wizard and all its options, see the [Template Wizard guide](xref:Uno.GettingStarted.UsingWizard).
```

File: doc/articles/create-an-app-rider.md [hunk 3]
```diff
@@ -12,17 +12,32 @@ uid: Uno.GettingStarted.CreateAnApp.Rider
...
 
-    ```bash
-    dotnet new unoapp --preset=blank -tfm net7.0 -o MyApp
-    ```
+    ![A screen showing the structure of the solution in Rider](Assets/quick-start/rider-folder-structure.png)
 
-    You should now have a folder structure that looks like this:  
-    ![rider-folder-structure](Assets/quick-start/rider-folder-structure.png)
+> [!TIP]
+> If you are not able to run the online App Wizard, you can explore the [`dotnet new` template](xref:Uno.GetStarted.dotnet-new) directly in the CLI.
```
Comment (Youssef1313) [inline doc/articles/create-an-app-rider.md:40]: I guess we may want to align on whether we use "online app wizard", "online wizard", or "live wizard" and keep that consistent when we refer to it.

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.1

File: doc/articles/get-started-dotnet-new.md [hunk 4]
```diff
@@ -25,6 +25,9 @@ To determine all parameters available for a template use `dotnet new [templatena
 
 This template can be used to create a multi-platform application for iOS, Android, WebAssembly, Mac Catalyst, Linux, and Win32 Desktop which uses the new WinUI 3 APIs.
 
+> [!TIP]
+> To create a new uno app and browse all the capabilities of the template, head to our [Live Wizard](https://aka.platform.uno/app-wizard) to create a `dotnet new` command line. For a detailed overview of the Uno Platform project wizard and all its options, see the [Wizard guide](xref:Uno.GettingStarted.UsingWizard).
```
Comment (ajpinedam) [inline doc/articles/get-started-dotnet-new.md:29]: should it be capital?
```suggestion
> To create a new Uno app and browse all the capabilities of the template, head to our [Live Wizard](https://aka.platform.uno/app-wizard) to create a `dotnet new` command line. For a detailed overview of the Uno Platform project wizard and all its options, see the [Wizard guide](xref:Uno.GettingStarted.UsingWizard).
```

**Review by ajpinedam — APPROVED** [review]

**Pull request merged**